gcloud-java-dns/src/test/java/com/google/gcloud/dns/DnsTest.java

```diff
@@ -33,33 +33,33 @@
   public void testDnsRecordListOption() {
     // dns name
     String dnsName = "some name";
-    Dns.DnsRecordListOption dnsRecordListOption = Dns.DnsRecordListOption.dnsName(dnsName);
-    assertEquals(dnsName, dnsRecordListOption.value());
-    assertEquals(DnsRpc.Option.NAME, dnsRecordListOption.rpcOption());
+    Dns.RecordSetListOption rrsetListOption = Dns.RecordSetListOption.dnsName(dnsName);
```

Comment:
Nit: my preference would be for calling this `recordSetListOption` (which is no longer than the previous `dnsRecordListOption`). I feel like `rrset` is not very human-readable.